src/Wallet.js

```diff
@@ -15,3 +15,12 @@ exports._getNamiCollateral = (nami) => () =>
 
 // _signTxNami :: String -> NamiConnection -> Effect (Promise String)
 exports._signTxNami = (txHex) => (nami) => () => nami.signTx(txHex);
+
+// _submitTxNami :: String -> NamiConnection -> Effect (Promise String)
+exports._submitTxNami = (txHex) => (nami) => () => {
```

Comment:
An aside: still learning JS but is there a reason for parameter parenthesis convention as we seem to have both? Not a request to change, just curious.

Reply:
No reason, I just have autoformat enabled. It's not necessary